index.js

```diff
@@ -135,7 +134,7 @@ GoogleContacts.prototype.getContact = function (cb, params) {
     function receivedContact(err, contact) {
         if (err) return cb(err);
 
-        cb(null, contact);
+        cb(null, _.get(contact, 'entry', []));
```

Comment:
to get the json object specific for a contact, still not as clean as when obtaining all contacts